psi/pmtdescriptor.go

```diff
@@ -43,6 +43,7 @@ const (
 	MAXIMUM_BITRATE    uint8 = 14  // 0000 1110 (0x0E)
 	AVC_VIDEO          uint8 = 40  // 0010 1000 (0x28)
 	STREAM_IDENTIFIER  uint8 = 82  // 0101 0010 (0x52)
+        TTML_SUBTITLING    uint8 = 127 // 0111 1111 (0x7F)
```

Comment:
Simple changes, but there's some formatting inconsistencies here and in similar spots in the changes made to `pmtstreamtype.go` . Can you run `gofmt` on this to get everything realigned?

Reply:
Done. Thanks for reviewing it!